packages/styling/mixins/table.scss

```diff
@@ -0,0 +1,210 @@
+  // inheritance to nested tables.
+  .dso-rich-content table > thead > tr,
+  .dso-rich-content table > tbody > tr,
+  .dso-rich-content table > tfoot > tr,
+  .table > thead > tr,
+  .table > tbody > tr,
+  .table > tfoot > tr {
+    > td.#{$state},
+    > th.#{$state},
+    &.#{$state} > td,
+    &.#{$state} > th {
+      background-color: $background;
+    }
+  }
+
+  // Hover states for `.table-hover`
+  // Note: this is not available for cells or rows within `thead` or `tfoot`.
+  .table-hover > tbody > tr {
+    > td.#{$state}:hover,
+    > th.#{$state}:hover,
+    &.#{$state}:hover > td,
+    &:hover > .#{$state},
+    &.#{$state}:hover > th {
+      background-color: $table-bg-active;
+    }
+
+    &:hover > th,
+    &:hover > td {
+      background-color: $table-bg-active;
+
+      a {
+        color: $dso-table-link-color;
+
+        &:hover {
+          text-decoration: none;
+        }
+      }
+    }
+  }
+
+  .table > tbody > tr {
+    &.#{$state} > th,
+    &.#{$state} > td {
+      background-color: $table-bg-active;
+
+      a {
+        color: $dso-table-link-color;
+
+        &:hover {
+          text-decoration: none;
+        }
+      }
+    }
+  }
+}
+
+@mixin dso-table() {
+  // Cells
+  margin-bottom: $line-height-computed;
+  max-width: 100%;
+  width: 100%;
+
+  &.dso-text-left {
+    th,
+    td {
+      text-align: left;
+    }
+  }
+
+  &.dso-text-center {
+    th,
+    td {
+      text-align: center;
+    }
+  }
+
+  &.dso-text-right {
+    th,
+    td {
+      text-align: right;
+    }
+  }
+
+  th,
+  td {
+    &.dso-text-left {
+      text-align: left;
+    }
+
+    &.dso-text-center {
+      text-align: center;
+    }
+
+    &.dso-text-right {
+      text-align: right;
+    }
+  }
+
+  > thead,
+  > tbody,
+  > tfoot {
+    > tr {
+      > th,
+      > td {
+        line-height: $line-height-base;
+        padding: ($table-cell-padding * 1.5) $table-cell-padding;
+        vertical-align: top;
+
+        a {
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+
+  > thead {
+    > tr {
+      > td,
+      > th {
+        border-bottom: 1px solid $dso-table-header-border-color;
+      }
+    }
+  }
+
+  > tbody,
+  > tfoot {
+    > tr {
+      > td,
+      > th {
+        border-bottom: 1px solid $table-border-color;
+      }
+    }
+  }
+
+  // Account for multiple tbody instances
+  > tbody + tbody {
+    border-top: 2px solid $table-border-color;
+  }
+
+  // Nesting
+  .table {
+    background-color: $body-bg;
+  }
+
+  // error after filtering
+  > tbody > tr.result-error {
+    td {
+      font-style: italic;
+    }
+  }
+
+  // sortable table
+  > thead > tr > th {
+    // Bottom align for column headings
+    vertical-align: bottom;
+
+    .btn.dso-sort {
+      color: $grijs-90;
+      font-weight: 600;
+
+      .di {
+        color: $grijs-60;
+
+        &.dso-sort-active {
+          color: $grijs-90;
+        }
+      }
+    }
+  }
+}
+
+@mixin dso-table-responsive {
```

Comment:
Huh? Dubbel!? Staat ook in components/table. Controleer even of deze file nog klopt

Reply:
Bestand verwijderd